pkg/controller/plan/migration.go

```diff
@@ -847,6 +854,39 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 			step.Phase = Completed
 			vm.Phase = r.next(vm.Phase)
 		}
+	case ConvertOpenstackSnapshot:
+		step, found := vm.FindStep(r.step(vm))
+		if !found {
+			vm.AddError(fmt.Sprintf("Step '%s' not found", r.step(vm)))
+			break
+		}
+		step.MarkStarted()
+		step.Phase = Running
+		pvcs, err := r.kubevirt.getPVCs(vm.Ref)
+		if err != nil {
+			r.Log.Error(err,
+				"Couldn't get VM's PVCs.",
+				"vm",
+				vm.String())
+			break
+		}
+
+		ready, err := r.builder.ConvertPVCs(pvcs)
+		if err != nil {
+			step.AddError(err.Error())
+			err = nil
+			break
+		}
+
+		if !ready {
+			r.Log.Info("Conversion isn't ready yet")
+			return err
```

Comment:
```suggestion
			return nil
```
or simply `return`? `return err` suggests that err might not be nil

Reply:
tried naked return originally but doesn't work because of the scoping of `err`